src/ch/blinkenlights/android/medialibrary/MediaLibrary.java

```diff
@@ -123,6 +124,7 @@ private static MediaLibraryBackend getBackend(Context context) {
 	 * @param context the context to use
 	 * @return MediaLibrary.Preferences
 	 */
+	@TargetApi(Build.VERSION_CODES.KITKAT)
```

Comment:
One of the methods below were only implemented in API KitKat, which is why Android studio added this tag.